pdns/dnsparser.cc

```diff
@@ -212,14 +212,15 @@ DNSRecord::DNSRecord(const DNSResourceRecord& rr): d_name(rr.qname)
 }
 
 // If you call this and you are not parsing a packet coming from a socket, you are doing it wrong.
-DNSResourceRecord DNSResourceRecord::fromWire(const DNSRecord& d) {
+DNSResourceRecord DNSResourceRecord::fromWire(const DNSRecord& record)
```

Comment:
how about calling `record` here `wire` instead? "record" is not more descriptive, and given both types involved are DNS[...]Record, it also doesn't distinguish anything